test/protobuf/protoc/proto/events/brex_events_elixirpb.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto2";
```

Comment:
Will put this file into `credit_card` repo. As described in the PR description, it works!